importexport/musicxml/musicxmlsupport.cpp

```diff
@@ -481,6 +481,10 @@ QString accidentalType2MxmlString(const AccidentalType type)
             case AccidentalType::SHARP_SLASH3:       s = "slash-quarter-sharp";  break;
             case AccidentalType::FLAT_SLASH2:        s = "double-slash-flat";    break;
             case AccidentalType::SHARP_SLASH2:       s = "slash-sharp";          break;
+            case AccidentalType::SHARP2_ARROW_DOWN:  s = "double-sharp-down";    break;
+            case AccidentalType::SHARP2_ARROW_UP:    s = "double-sharp-up";      break;
+            case AccidentalType::FLAT2_ARROW_DOWN:   s = "flat-flat-down";       break;
+            case AccidentalType::FLAT2_ARROW_UP:     s = "flat-flat-up";         break;
             case AccidentalType::SORI:               s = "sori";                 break;
             case AccidentalType::KORON:              s = "koron";                break;
             default:
```

Comment:
Maybe the default should be `s = "other"`? Maybe in addition to the `qDebug()` message, maybe without?

Reply:
Don't think so. The `other` value is supposed to go together with the `@smufl`attribute. 
See the answer bei @mdgood here: https://github.com/w3c/musicxml/issues/263

Reply:
OK, then let's just add that?
Probably something for a different PR though